go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -420,9 +420,9 @@ func runOvnKube(ctx context.Context, runMode *ovnkubeRunMode, ovnClientset *util
 	var masterWatchFactory *factory.WatchFactory
 	var err error
 
-	if runMode.networkControllerManager {
+	if runMode.ovnkubeController {
 		// create factory and start the controllers asked for
-		masterWatchFactory, err = factory.NewMasterWatchFactory(ovnClientset.GetMasterClientset())
+		masterWatchFactory, err = factory.NewNCMWatchFactory(ovnClientset.GetNetworkControllerManagerClientset())
```

Comment:
`NewControllerWatchFactory` and `GetControllerClientset`?
`NewKubeControllerWatchFactory` and `GetKubeControllerClientset`?
`NewOVNKubeControllerWatchFactory` and `GetOVNKubeControllerClientset`?

Reply:
yes i like your suggestions.. changing them

Reply:
done.